test/tstKokkosToolsDistributedAnnotations.cpp

```diff
@@ -38,6 +38,7 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(
         std::regex re("^(Testing::"
                       "|ArborX::DistributedTree::"
                       "|ArborX::BVH::"
+                      "|Kokkos::SortImpl::BinSortFunctor::"
```

Comment:
Where does that get introduced?

Reply:
I did not follow it up, just fixed it, and it makes sense.